docs/UserGuide.md

```diff
@@ -112,9 +115,16 @@ Adds a flashcard.
 Format: `add q/QUESTION a/ANSWER [c/CATEGORY] [r/RATING] [n/NOTE] [d/DIAGRAM] [t/TAG]...`
 
 * `RATING` must be a number between 1 and 5 inclusive.
-* `DIAGRAM` can be defined by a valid relative or absolute path.
 * `CATEGORY` must be alphanumeric and have a maximum of 50 characters. 
      It can consist of multiple words but there should only be 1 space between words.
+* `DIAGRAM` can be defined by a valid relative or absolute path.
+* `DIAGRAM` only supports file path without spaces in it. Unintended behaviour may occur if file path with spaces is used.
+  * d/images/diagram.png is supported.
+  * d/image s/diagram.png is not supported.
+* `DIAGRAM` only supports the following file types. Unintended behaviour may occur if file path with spaces is used.
+  * jpeg
+  * png
+  * bmp
```

Comment:
Is it possible to structure this as one paragraph? `Diagram` ..... instead of 3 separate chunks.
maybe use eg of d/images d/diagram.png instead for unsupported.
for the file types, no need write "unintended behavior". Remember to add jpg, gif also.